### PR TITLE
Fix focus loss project name

### PR DESCRIPTION
**Pull request by AlexandreSi**
Resolves https://forum.gdevelop-app.com/t/blank-project-name-issue/37558

### CODE REVIEW

File: newIDE/app/src/UI/SearchBar.js [hunk 1]
```diff
@@ -205,13 +206,20 @@ const SearchBar = React.forwardRef<Props, SearchBarInterface>(
         if (
           shouldAutofocusSearchbar &&
           tagsHandler &&
-          tagsHandler.chosenTags.length === 0
+          tagsHandler.chosenTags.length === 0 &&
+          previousChosenTagsCount.current > 0
         )
           focus();
       },
       [tagsHandler, shouldAutofocusSearchbar]
```
Comment (4ian) [inline newIDE/app/src/UI/SearchBar.js:214]: This may indicate also an issue where the tagsHandler was not stable between re-renders?

**Review by 4ian — APPROVED** [review]

File: newIDE/app/src/UI/SearchBar.js (hunk 1 above)
Comment (AlexandreSi) [reply newIDE/app/src/UI/SearchBar.js:214]: Yes you're right. setting the `tagsHandler` to:
```
const tagsHandler = React.useMemo(() => ({
    add: filtersState.addFilter,
    remove: filtersState.removeFilter,
    chosenTags: Array.from(filtersState.chosenFilters),
  }), [filtersState.chosenFilters])
```
fixes the issue. I think I'll add both as it is safer.

Comment (AlexandreSi) [reply newIDE/app/src/UI/SearchBar.js:214]: After thinking it through, I don't know if it's correct adding it.

Comment (AlexandreSi) [reply newIDE/app/src/UI/SearchBar.js:214]: After thinking it through again, I guess memoizing things is never bad

Comment (AlexandreSi) [reply newIDE/app/src/UI/SearchBar.js:214]: But indeed, tagsHandler seemed to change, but I didn't find why

File: newIDE/app/src/AssetStore/ExampleStore/index.js [hunk 2]
```diff
@@ -39,13 +43,33 @@ export const ExampleStore = ({ isOpening, onOpen }: Props) => {
...
+    [shouldAutofocusSearchbar, focusOnMount]
+  );
+
+  const tagsHandler = React.useMemo(
+    () => ({
+      add: filtersState.addFilter,
+      remove: filtersState.removeFilter,
+      chosenTags: Array.from(filtersState.chosenFilters),
+    }),
+    [filtersState]
```
Comment (4ian) [inline newIDE/app/src/AssetStore/ExampleStore/index.js:70]: A bit nervous about this useMemo, but I guess if addFilter/removeFilter/chosenFilters change, then filtersState change so it should be correct.

File: newIDE/app/src/AssetStore/ExtensionStore/index.js [hunk 3]
```diff
@@ -61,6 +61,15 @@ export const ExtensionStore = ({
       )
     : null;
 
+  const tagsHandler = React.useMemo(
+    () => ({
+      add: filtersState.addFilter,
+      remove: filtersState.removeFilter,
+      chosenTags: Array.from(filtersState.chosenFilters),
+    }),
+    [filtersState]
```
Comment (4ian) [inline newIDE/app/src/AssetStore/ExtensionStore/index.js:70]: (ditto)

File: newIDE/app/src/ProjectCreation/services/BrowserCreation.js [hunk 4]
```diff
@@ -12,16 +12,14 @@ import {
...
 const gd: libGDevelop = global.gd;
 
-// Signatures of this function and its arguments contain useless arguments
-// because this function has to match the signature of LocalCreation.onCreateBlank
-// that needs these arguments.
 export const onCreateBlank: OnCreateBlankFunction = async ({
   i18n,
-  outputPath,
-  projectName,
+  settings,
```
Comment (4ian) [inline newIDE/app/src/ProjectCreation/services/BrowserCreation.js:17]: Way cleaner 👍

**Review by 4ian — APPROVED** [review]
LGTM!

**Pull request merged**